components/soc/esp32/include/soc/soc_caps.h

```diff
@@ -116,7 +116,7 @@
 // control pullup/down/capability, independent with RTC registers.
 
 // 0~39 except from 20, 24, 28~31 are valid
```

Comment:
May want to update the comment here to reflect the change made below -
```suggestion
// 0~39 except for 24, 28~31 are valid
```